pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -786,18 +786,16 @@ type BackupSpec struct {
 // +k8s:openapi-gen=true
 // BRConfig contains config for BR
 type BRConfig struct {
-	// PDAddress is the PD address of the tidb cluster
-	PDAddress string `json:"pd"`
+	// Whether enable TLS in TiDBCluster
+	EnableTLSClient bool `json:"enableTLSClient,omitempty"`
+	// ClusterName of backup/restore cluster
+	Cluster string `json:"cluster,omitempty"`
```

Comment:
This cannot be empty, right?

Reply:
yes，I have verify it in the validate function